src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,13 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.7.0
+TBD
+
+### Features
+
+- Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
```

Comment:
@omnomagonz what is the term for the UI? Okteto UI or Dashboard? I've seen both in the docs 😛

Reply:
If we agree to that term sounds good. But then we'll need someone to go over the docs and make sure this is consistent 😀. Can you drive that?

Reply:
Sure (what did I dooooooooooooo!?)

It likely aligns well with my existing effort to review docs so I'll make a note to check for this language 👍

Reply:
yeah, I figured we can bundle this on that effort. It doesn't have to be as part of this PR